api/python/notebooks/analysis_demo/comp_bio_scvi_model_use.ipynb

```diff
@@ -122,7 +122,7 @@
     }
    ],
    "source": [
-    "!aws s3 cp s3://cellxgene-contrib-public/models/scvi/2023-12-15/homo_sapiens/model.pt 2023-12-15-scvi-homo-sapiens/scvi.model/"
+    "!aws s3 sync --no-sign-request  --no-progress --only-show-errors s3://cellxgene-contrib-public/models/scvi/2023-12-15/homo_sapiens/model.pt 2023-12-15-scvi-homo-sapiens/scvi.model/"
```

Comment:
I think you need to keep `cp` here. I tried and `sync` doesn't work since it's trying to copy a specific file.